code/datums/brain_damage/mild.dm

```diff
@@ -54,7 +54,7 @@
 	..()
 
 /datum/brain_trauma/mild/dumbness/on_lose()
-	owner.remove_disability(DUMB, TRAUMA_DISABILITY)
+	owner.remove_disability(DUMB, list(TRAUMA_DISABILITY))
```

Comment:
undo these for remove_disability to keep them consistent with add_disability.

Reply:
Alright